less/variables.less

```diff
@@ -944,19 +952,19 @@
 
 @panel-success-text:          @state-success-text;
 @panel-success-border:        @state-success-border;
-@panel-success-heading-bg:    #FFF;         //@state-success-bg;
+@panel-success-heading-bg:    #FFF;
 
 @panel-info-text:             @state-info-text;
 @panel-info-border:           @state-info-border;
-@panel-info-heading-bg:       #FFF;         //@state-info-bg;
+@panel-info-heading-bg:       #FFF;
```

Comment:
There is one actually, @brand-white, in case we ever went for an off white, was just tidying this comment up, would prefer to make those updates another time.